src/robusta/cli/main.py

```diff
@@ -172,6 +172,7 @@ def gen_config(
                 ms_teams_sink=MsTeamsSinkParams(
                     name="main_ms_teams_sink",
                     webhook_url=msteams_webhook,
+                    use_emoji=False
```

Comment:
I think this isn't required if you put default value on the pydantic class
Also, this will only effect new clients.
We want it to work to all existing clients